src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -62,6 +62,11 @@ internal abstract class PENamedTypeSymbol : NamedTypeSymbol
         /// </summary>
         private TypeKind _lazyKind;
 
+        /// <summary>
+        /// Lazily initiated HasEmbeddedAttribute
+        /// </summary>
+        private ThreeState _lazyHasEmbeddedAttribute = ThreeState.Unknown;
```

Comment:
Should this be part of `UncommonProperties` rather than a field directly on `PENamedTypeSymbol`?